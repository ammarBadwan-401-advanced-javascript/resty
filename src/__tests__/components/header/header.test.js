import React from 'react';
import { shallow, mount} from 'enzyme';
import renderer from 'react-test-renderer';

import Header from '../../../components/header/header';

describe('Form test',()=>{
  it('Buttons exist in the page',()=>{
    let app = shallow(<Header/>);
    expect(app.find('header').exists()).toBeTruthy();
  });

  it('renders correctly', ()=> {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot();
  });

});
