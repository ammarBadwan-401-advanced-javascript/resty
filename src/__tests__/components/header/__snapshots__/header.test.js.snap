// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form test renders correctly 1`] = `
<header>
  <h1>
    RESTY PROJECT
  </h1>
</header>
`;
